spec/support/define_methods_for_testing_memo_wise.rb

```diff
@@ -33,7 +33,7 @@ def self.define_methods_for_testing_memo_wise(target:, via:)
     self_kw_arg = via == :self_dot ? "self: " : ""
     self_class_method = via == :self_dot ? "_class_method" : ""
 
-    target.module_eval <<~END_OF_METHOD, __FILE__, __LINE__ + 1
+    target.module_eval <<~END_OF_METHODS, __FILE__, __LINE__ + 1
```

Comment:
Unrelated, but this made more sense to me since there are multiple methods defined here.